### PR TITLE
Admin Quick Leveler and Easy Gift Extractor

### PR DESCRIPTION
**Pull request by InsightfulParasite**
## About The Pull Request
Small PR adding quality of life tool for abnormality coders. A quick attribute leveler that gives you 100 stats in all attributes and a easy gift granter that you can just use on a abnormality, breached or unbreached.
Due to this change being only one file i also added some fixes to gadgets so they no longer just not work.

## Why It's Good For The Game
When i was coding snow whites apple and adding the special effect for her gift it was problematic to jump through all these hoops. So i made some shortcuts for other coders to ease the tediousness.

## Changelog
:cl:
add: /obj/item/lc13_attribute_tester & /obj/item/lc13_easygift_tester, just find them by searching test.
fix: Ordeal detector, Abnormality detector reaction to detecting a aleph, Deep Scanner
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
Honestly, abnormality coders should know how to do it all on their own and I was going to suggest moving it to a proc/admin verb, but this seems fine.

**Comment by EgorDinamit** [comment]
Just squash commits and this should be good to go.

**Comment by Kitsunemitsu** [comment]
The problem is having to change my stats every time I test ego weapons.... which is sometimes 5-6x a session and turns a 3 minute test into a 4 minute test

File: ModularTegustation/tegu_items/lc13unique_items.dm [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+	user.adjust_all_attribute_levels(100)
+	qdel(src)
+
+/obj/item/lc13_easygift_tester
+	name = "gift extractor"
+	desc = "Unpopular due to its excessive energy use, this device extracts gifts from a entity on demand."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "hammeroff"
+
+/obj/item/lc13_easygift_tester/attack(mob/living/simple_animal/hostile/abnormality/M, mob/living/carbon/human/user)
```
Comment (EgorDinamit) [inline ModularTegustation/tegu_items/lc13unique_items.dm:19]: You forgot to add a check for if the mob is an abnormality type at all.

Comment (InsightfulParasite) [reply ModularTegustation/tegu_items/lc13unique_items.dm:19]: I added a isabnormality(M)

**Review by Kitsunemitsu — APPROVED** [review]

**Pull request merged**